lms/djangoapps/discussion_api/urls.py

```diff
@@ -0,0 +1,17 @@
+"""
+Discussion API URLs
+"""
+from django.conf import settings
+from django.conf.urls import patterns, url
+
+from discussion_api.views import CourseTopicsView
+
+
+urlpatterns = patterns(
+    "discussion_api",
+    url(
+        r"^v1/course_topics/{}".format(settings.COURSE_ID_PATTERN),
```

Comment:
No; single and double quotes are equivalent, and I prefer to use double quotes throughout.